src/components/Map/index.tsx

```diff
@@ -44,6 +52,7 @@ export const transitionProps = {
   transitionEasing: easeInOutQuad,
 }
 
+const MAX_ZOOM = 17
```

Comment:
A few lines below `MAP_CONFIG.maxZoom` is defined. Perhaps we should unify the values, unless there is a reason for why they are different?